src/main/java/jenkins/plugins/publish_over_ftp/BapFtpHostConfiguration.java

```diff
@@ -60,16 +69,22 @@
     private final String controlEncoding;
     private final boolean disableMakeNestedDirs;
     private final boolean disableRemoteVerification;
+    private final boolean useFtpOverTls;
+    private final String trustedCertificate;
+
     @DataBoundConstructor
     public BapFtpHostConfiguration(final String name, final String hostname, final String username, final String encryptedPassword,
                                    final String remoteRootDir, final int port, final int timeout, final boolean useActiveData,
-                                   final String controlEncoding, final boolean disableMakeNestedDirs, boolean disableRemoteVerification) {
+                                   final String controlEncoding, final boolean disableMakeNestedDirs, boolean disableRemoteVerification,
+                                   final boolean useFtpOverTls, final String trustedCertificate) {
```

Comment:
This will break people who are using the previous constructor. You may want to use @DataBoundSetter instead to maintain compatibility.